viper/modules/pehash/pehasher.py

```diff
@@ -6,30 +6,11 @@
 
 import bz2
 import hashlib
-
-try:
```

Comment:
Having try-except clauses all over the code is expected, as not al the users want to install *all* the dependencies.